pricing-page-exercise/mesut/index.html

```diff
@@ -1,12 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML 5 Boilerplate</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/5e08b15075.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="company-name"><a href="#" class="company-link link">Company name</a></div>
+    <div class="top-right-links">
+      <ul class="top-right-buttons">
```

Comment:
I always say do not select elements directly but in this case it's ok not to add a class to `ul` element in the navbar. You can do something like:
`.navbar-right > ul`.
That's because we usually only have one navbar and in this navbar there's only one list.